site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,552 @@
+* AutoAPI, which bridges .NET and Sphinx - https://sphinx-autoapi.readthedocs.io/en/latest/
+
+Tables
+------
+
+Working with tables can be the bane of anyone who uses plaintext markup languages.
+Most other languages require that you write them in the file with some arcane syntax.
+However with reStructuredText,
+you can use directives to make this much easier.
+
+You can use either of two powerful list directives,
+``csv-table`` and ``list-table``.
+Here is an example of ``csv-table``::
+
+    .. csv-table:: Frozen Delights!
+       :header: "Treat", "Quantity", "Description"
+       :widths: 15, 10, 30
+
+       "Albatross", 2.99, "On a stick!"
+       "Popcorn", 1.99, "Straight from the oven"
+
+This shows the inline markup,
+however the CSV can also be managed in an external file.
+This allows you to manage your complex tables in a third party tool,
+and have your documentation consume them from a CSV which is a much nicer workflow.
+
+Internationalization
+--------------------
+
+Sphinx includes support for translating documentation into multiple languages.
+Since sphinx knows the structure of your documents,
+it is able to generate a catalog of content split by each paragraph.
+
+Sphinx internationalization works using the *gettext* system.
+It ships with a builder that allows you to generate a catalog::
+
+    make gettext
+
+You can then use that catalog as the base translation,
+using any tool that supports gettext.
+I recommend using Transifex,
+which gives you a web-based system for translating the documentation:
+
+.. image:: transifex.png
+
+You can then tell Sphinx what language to generate for its documentation when you build it by setting the ``language`` setting.
+Read the Docs also supports translation,
+allowing you to specify the language of the documentation.
+
+More information on Sphinx internationalization can be found here:
+
+* http://www.sphinx-doc.org/en/stable/intl.html
+* http://docs.readthedocs.io/en/latest/localization.html
+
+My blog
+-------
+
+Sphinx is quite versatile,
+which means you can use it for lots of different things.
+I use a package called ``ablog`` for hosting of my blog over at http://ericholscher.com.
+
+The simplest thing you do is specify that a document is a blog post with the ``post`` directive::
+
+    .. post:: 2016-03-15 09:00
+       :tags: writing, stc, sphinx
+
+This adds it to the set of posts that get shown in any document where you use::
+
+    .. postlist:: 
+       :tags: stc
+
+You can even create specific archive pages for categories or tags with ``postlist``,
+or just output all of your posts.
+
+This shows some of the magical things you can do with Sphinx's extensibility.
+If you're curious,
+this article was actually written in reStructuredText and then exported to Word for publishing.
+You can see the RST in full here: https://github.com/ericholscher/ericholscher.com/blob/master/site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst
```

Comment:
If you're looking for something to cut, I'd condense this down. It's a great way to plug the blog though :) You keep the thought be condensing this down to a "even my blog, where you might be reading this, is powered by sphinx" sentiment.